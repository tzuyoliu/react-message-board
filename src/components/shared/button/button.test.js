import React from 'react'
import { create } from 'react-test-renderer'
import Button from '~/components/shared/button'

describe('button.test.js', () => {
    test('<Button /> should render', () => {
        const button = create(<Button />)
        expect(button).toBeTruthy()
    })

    test('<Button /> should match the snapshot', () => {
        const button = create(<Button>Snapshot Test</Button>)
        expect(button.toJSON()).toMatchSnapshot()
    })
})
