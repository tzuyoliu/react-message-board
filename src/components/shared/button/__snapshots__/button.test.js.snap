// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button.test.js <Button /> should match the snapshot 1`] = `
<button
  className="sc-bdVaJa kktxWd"
>
  Snapshot Test
</button>
`;
